contrib/core/actions/stormbot_say/stormbot_say

```diff
@@ -9,17 +9,20 @@ st2_endpoints = {
     'datastore': 'http://localhost:9101'
 }
 
+def getRequiredKey( client, key_name ) :
```

Comment:
Minor style thing - please use underscore separated names for function names and no spaces after parenthesis. I will fix this after merge.